src/endpoints/account-memberships.js

```diff
@@ -10,6 +11,27 @@ class AccountMembershipsEndpoint extends BaseExtend {
     )
   }
 
+  All(accountId, token = null, headers = {}) {
```

Comment:
You removed token and headers from definitions but they are still here in javascript.